src/index.html

```diff
@@ -11,7 +11,146 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+      onsubmit="onSubmit()">
```

Comment:
Read this message
https://mateacademy.slack.com/archives/C02CSGP1KD1/p1630943541196100?thread_ts=1630943169.192800&cid=C02CSGP1KD1
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      autocomplete="off"
      onsubmit="onSubmit()"
    >
```